content/en/apps/examples/anc.md

```diff
@@ -79,21 +86,141 @@ Both maternal and newborn care workflows are defined to connect form actions and
+        resolution="The CHW submits a child treatment follow up confirming the patient’s health status." 
+        %}}
+  {{% workflow
+         condition="The child assessment form is submitted with a fever that lasts less than 7 days and with a RDT that is negative or RDT not done."
+         task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form indicates that the patient has a cough less than 14 days."
+         task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form indicates that the patient has a cough less than 14 days and a fast breathing."
+         task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+   {{% workflow
+         condition="The child assessment form is submitted with a diarrhea that lasts less than 14 days and with no blood."
+         task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+         %}}
+  {{% workflow
+         condition="The child assessment form is submitted with a yellow MUAC."
+         task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
+         resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
+  %}}
```

Comment:
We should coalesce the tasks:

```suggestion
  {{% workflow
        condition="A child assessment form is submitted with any of the following: 
          - danger signs symptoms
          - fever that lasts 7+ days
          - cough that lasts 14+ days
          - fast breathing (50+ bpm, 40+ for infants)
          - specific diarrhea symptoms"
        task="A [child referral follow up task](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.gwiatej2wjp) appears 3 days after the assessment, and lasts for 3 days." 
        resolution="The CHW submits a child referral follow up form confirming that the patient has attended the clinic visit."
  %}}

  {{% workflow
        condition="A child assessment form doesn’t trigger a referral, yet has one of the following condition: 
          - fever that lasts less than 7 days and RDT that is positive
          - fever that lasts less than 7 days and with a RDT that is negative or RDT not done
          - cough less than 14 days, without fast breathing
          - diarrhea that lasts less than 14 days and with no blood
          - yellow MUAC"
        task="Two [child treatment follow up tasks](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.9sln0fedxe4x) are triggered. The first one 3 days and the second one 6 days after the reported date. The tasks should stay for 3 days after the due date."
        resolution="The CHW submits a child treatment follow up confirming the patient’s health status."
  %}}

  {{% workflow
        condition="A child assessment form doesn’t trigger a referral or treatment, yet has a red MUAC."
        task="A [malnutrition follow up task](https://docs.google.com/document/d/1hC7gDsXxbjtoob2g36HNeuAReoWJDP2ffjQe7m6O-uY/edit#heading=h.u5ijb7zi3ei1) appears after 3 days. The task persists 3 days after the due date."
        resolution="The CHW submits a malnutrition follow up form confirming that the patient has attended the clinic visit." 
  %}} 
```

Reply:
fixed in a080c3090ba86dcf572f5dfa98e1c1b4e3a48534